applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -667,6 +676,7 @@ static void fix_rep_wk(struct k_work *work)
 		/* help nrf_cloud_pgps as most recent known location */
 		nrf_cloud_pgps_set_location(pvt.latitude, pvt.longitude);
 	}
+#endif
```

Comment:
If CONFIG_SLM_PGPS is not defined, this implementation causes empty label (update_pgps) and generates a compiler error "../src/gnss/slm_at_gnss.c:662:1: error: label at end of compound statement"
You can even add 
#else
    ;
#endif

 Or just add "return;" at the end of the function.

Otherwise, the PR works, tested via SLM tests.

Reply:
Fixed.